src/main/java/com/xpandit/plugins/xrayjenkins/model/ServerConfiguration.java

```diff
@@ -122,7 +112,7 @@ public FormValidation doTestConnection(@AncestorInPath final Item item,
 
         Jenkins.getInstance().checkPermission(Jenkins.ADMINISTER);
         if (StringUtils.isBlank(credentialId)) {
-            return FormValidation.error("Authentication not filled!");
+            return FormValidation.error("Authentication is Optional, however it's required in order to test the connection.");
```

Comment:
Authentication is optional, however it's required in order to test the connection.